lib/spack/spack/config.py

```diff
@@ -439,10 +440,10 @@ def get_config(self, section, scope=None):
             scopes = [self._validate_scope(scope)]
 
         merged_section = syaml.syaml_dict()
-        for scope in scopes:
+        for scope_ in scopes:
```

Comment:
why does this have an underscore?

Reply:
To distinguish it from the input of the same name.  Will change per offline discussion.

Reply:
Removing